workLoggingSystem.html

```diff
@@ -295,7 +295,7 @@
                     <div class="row formComponent">
                         <div class="col">
                             <div class="row">
-                                <div class="col-md-4 menu">Estimated Time:</div>
+                                <div class="col-md-4 menu" id="ttt">Estimated Time:</div>
```

Comment:
No need to give them id, You can directly change the text color of input tag id's to red, will reduce the complexity of the code